queries/php_only/highlights.scm

```diff
@@ -1,3 +1,170 @@
+; Keywords
+[
+  "and"
+  "as"
+  "instanceof"
+  "or"
+  "xor"
+] @keyword.operator
+
+[
+  "fn"
+  "function"
+] @keyword.function
+
+[
+  "break"
```

Comment:
break is like `break` in for, while loop, is it correct? If yes, please use `keyword.repeat` for it, just like other languages

Reply:
I can do that!